rule-packs/sophos-endpoint-security

```diff
@@ -0,0 +1,166 @@
+#start-sophos-endpoint-misconfigurations
+[
+  {
+    "name": "sophos-misconfigured-endpoints-1",
+    "description": "This will query for endpoints that do not have sophos edr installed.",
+    "queries": [
+      {
+        "name": "query0",
+        "query": "FIND Device !PROTECTS << sophos_endpoint",
+        "version": "v1"
+      }
+    ]
+    "alertLevel": "MEDIUM"
+  },
+  {
+    "name": "sophos-misconfigured-endpoints-2",
+    "description": "This will query for endpoints that are missing a relationship with a user. This will help determine potential rogue devices",
+    "queries": [
+      {
+        "name": "query0",
+        "query": "Find sophos_sensor that !HAS User THAT IS Person THAT IS okta_user|azure_user|google_user",
+        "version": "v1"
+      }
+    ]
+    "alertLevel": "MEDIUM"
+  },
+  {
+    "name": "sophos-misconfigured-endpoints-3",
+    "description": "This query is looking for devices that do not have tamper-protection enabled. Tamper Protection is a feature that prevents unauthorized users and certain types of known malware from uninstalling Sophos security software or turning it off through the Sophos interface.",
+    "queries": [
+      {
+        "name": "query0",
+        "query": "FIND sophos_endpoint with tamperProtectionEnabled != true ",
+        "version": "v1"
+      }
+    ]
+    "alertLevel": "MEDIUM"
+  },
+  {
+    "name": "sophos-misconfigured-endpoints-4",
+    "description": "This query is looking for endpoints with an unacceptable lockdown status. Any status locked, notInstalled, unavailable,uninstalled is considered unacceptable",
+    "queries": [
+      {
+        "name": "query0",
+        "query": "FIND sophos_endpoint with lockdownStatus != ("installing" or "registering" or "starting" or "stopping" or "unlocked")",
+        "version": "v1"
+      }
+    ]
+    "alertLevel": "MEDIUM"
+  },
+  {
+    "name": "sophos-misconfigured-endpoints-5",
+    "description": "This query is looking for endpoints with an unacceptable update status. This includes any endpoint with a status of notInstalled.",
+    "queries": [
+      {
+        "name": "query0",
+        "query": "FIND sophos_endpoint with updateStatus != ("upToDate" or "updating" or "rebootRequired")",
+        "version": "v1"
+      }
+    ]
+    "alertLevel": "MEDIUM"
+  },
+  {
+    "name": "sophos-misconfigured-endpoints-6",
+    "description": "This query will look for endpoints with misconfigured ports. This will help in determining potential rogue devices or devices of risk.",
+    "queries": [
+      {
+        "name": "query0",
+        "query": "FIND sophos_device with portsEnabled != "443"",
+        "version": "v1"
+      }
+    ]
+    "alertLevel": "MEDIUM"
+  },
+  {
+    "name": "domain-expires-in-30-days",
```

Comment:
does this rule belong in the Sophos pack? looks unrelated

Reply:
deleted